auth_token/utils.py

```diff
@@ -46,9 +51,13 @@ def login(request, user, expiration=True, auth_slug=None, related_objs=None, bac
                                  expiration=expiration, auth_slug=auth_slug, ip=get_ip(request),
                                  backend=backend, allowed_cookie=allowed_cookie, allowed_header=allowed_header)
 
+    if not two_factor_login:
+        token.is_authenticated = True
```

Comment:
Dont know if this is OK, you are basically saying that this token was verified using second factor, which is not true?....

Reply:
token.is_authenticated = not two_factor_login ?

Reply:
Yes, this is true.

Reply:
fix it please :)

Reply:
```

token = Token.objects.create(
  user=user, user_agent=request.META.get('HTTP_USER_AGENT', '')[:256],
  expiration=expiration, auth_slug=auth_slug, ip=get_ip(request),
  expiration=expiration, auth_slug=auth_slug, ip=get_ip(request),
   backend=backend, allowed_cookie=allowed_cookie, allowed_header=allowed_header)	                                 
  backend=backend, allowed_cookie=allowed_cookie, allowed_header=allowed_header, 
  is_authenticated=not two_factor_login
)
```